### DISCUSSION TITLE
Install peer dependencies from npm CLI

### DISCUSSION THREAD

**Discussion by AlanBlanchet** [npm/rfcs#654] [RRFC] [answered]
Hello community,

I was trying to install a peer dependency from the CLI and I wasn't able to.
Is there something I am missing ?

Could we add an option to add a peer dependency to this already existing list ?
`[-P|--save-prod|-D|--save-dev|-O|--save-optional] [-E|--save-exact] [-B|--save-bundle] [--no-save] [--dry-run]`.

A `--save-peer` would be appropriate ? Seeing that the `-P` shortcut is already used I don't really know which one could be used for peers. Maybe `-PE` ? `-X` as in "external" ?

Thank you for reading this.

**Comment by ruyadorno** [comment, marked as answer]
[`--save-peer`](https://docs.npmjs.com/cli/v9/using-npm/config#save-peer) is already an existing option and can be used for that.

**Reply by AlanBlanchet** [reply]
I really tried searching for it and couldn't find anything...
Sorry for wasting you time and thanks for your reply.

**Reply by AlanBlanchet** [reply]
To anyone coming across this discussion. Make sure you have npm version >= 7 installed. I was on 6.x.x
